Airports/EGVP/Runway.txt

```diff
@@ -1,2 +1,3 @@
+04  22  040 220 N051.08.54.810 W001.33.35.650 N051.09.02.150 W001.33.25.730
```

Comment:
```suggestion
04  22  041 221 N051.08.54.810 W001.33.35.650 N051.09.02.150 W001.33.25.730
```
We usually use the magnetic heading, so the two values should be rounded up as per AD 2.16